static/css/impala/site.less

```diff
@@ -10,6 +10,7 @@ html, body {
 
 #main-wrapper {
     background: #fff url(../../img/zamboni/global/bg-header.png) repeat-x;
+    color: #000;
```

Comment:
I reckon the body should ideally have a background color assuming it doesn't cause any unexpected regressions.